python/requirements.in

```diff
@@ -6,6 +6,7 @@ email-validator
 fastapi
 healpy
 mangum
+python-multipart
```

Comment:
I don't see any new imports related to this package.

Reply:
It's required for FastAPI to support multi-part POST, e.g.. uploading of a file.

Reply:
```suggestion
python-multipart  # Required for file uploads in FastAPI; see https://fastapi.tiangolo.com/tutorial/request-files/
```